source/jobs/_contact_us_no_roles.erb

```diff
@@ -0,0 +1,23 @@
+<% roles_count ||= 1 %>
```

Comment:
I don't think this line is necessary as we're not using that variable anywhere here in the code

Reply:
Good point, I missed that one when I copied from the other file. Fixing it now.